integ-tests/src/main/java/com/google/testdata/TestDataHelper.java

```diff
@@ -28,12 +28,15 @@ private TestDataHelper() {}
    */
   public static void generateUsers(Connection connection, int userCount) throws SQLException {
     final String sqlDataPath = "src/main/java/com/google/sql/data/users_data.sql";
+    final String password = "test_" + nanoTime();
```

Comment:
Should we add a random postfix to make it impossible to guess?

Reply:
Using `UUID.randomUUID()` is probably the easiest here w/o adding a dependency to apache commons, which has a nice util method for this.

Reply:
Done.